tox.ini

```diff
@@ -4,17 +4,12 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26,py27,cover,lint
+envlist = py26,py27,lint
```

Comment:
The reasons why I put cover into same run as normal testing are:
a) less waiting for tests with coverage to complete, guess it is about -25% if it is done within a normal test run and not needed a whole separate test run just for coverage
b) it might get executed more often if it is part of normal test run and people do not need to invoke it separately. so people might actually look at it more often and have less test commands to invoke (if not using tox)
c) as it is now, it is just showing covery, so no breaking anything AFAIK